templates/Element/FilterBox/filter_box_common.twig

```diff
@@ -123,6 +125,8 @@
                         label=""
                         @change="onCategoryChange"
                     ></category-picker>
+                {% else %}
+                    -
```

Comment:
Non ho capito bene cos'è questo "-" (appare effettivamente in pagina?) però a meno che non debba apparire e sia voluto, rimuoverei l'else

Reply:
appare in pagina. casistica in cui nessuno dei due casi prima è attivo (ma è presente la label "Categorie"). Un filtro vuoto, di fatto. Complesso rimuovere la label a monte, perché è un filtro dinamico.